Projects/Checkers/Board.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace Checkers;
+
+public class Board
+{
+	public List<Piece> Pieces { get; set; }
+
+	public Piece? Aggressor { get; set; }
+
+	public Piece? this[int x, int y] =>
+		Pieces.FirstOrDefault(piece => piece.X == x && piece.Y == y);
+
+	public Board()
+	{
+		Aggressor = null;
+		Pieces = new List<Piece>
+			{
+				new() { NotationPosition ="A3", Color = Black},
+				new() { NotationPosition ="A1", Color = Black},
+				new() { NotationPosition ="B2", Color = Black},
+				new() { NotationPosition ="C3", Color = Black},
+				new() { NotationPosition ="C1", Color = Black},
+				new() { NotationPosition ="D2", Color = Black},
+				new() { NotationPosition ="E3", Color = Black},
+				new() { NotationPosition ="E1", Color = Black},
+				new() { NotationPosition ="F2", Color = Black},
+				new() { NotationPosition ="G3", Color = Black},
+				new() { NotationPosition ="G1", Color = Black},
+				new() { NotationPosition ="H2", Color = Black},
+
+				new() { NotationPosition ="A7", Color = White},
+				new() { NotationPosition ="B8", Color = White},
+				new() { NotationPosition ="B6", Color = White},
+				new() { NotationPosition ="C7", Color = White},
+				new() { NotationPosition ="D8", Color = White},
+				new() { NotationPosition ="D6", Color = White},
+				new() { NotationPosition ="E7", Color = White},
+				new() { NotationPosition ="F8", Color = White},
+				new() { NotationPosition ="F6", Color = White},
+				new() { NotationPosition ="G7", Color = White},
+				new() { NotationPosition ="H8", Color = White},
+				new() { NotationPosition ="H6", Color = White}
+			};
+	}
+
+	public static string ToPositionNotationString(int x, int y)
+	{
+		if (!IsValidPosition(x, y)) throw new ArgumentException("Not a valid position!");
+		return $"{(char)('A' + x)}{y + 1}";
+	}
+
+	public static (int X, int Y) ParsePositionNotation(string notation)
+	{
+		if (notation is null) throw new ArgumentNullException(nameof(notation));
+		notation = notation.Trim().ToUpper();
+		if (notation.Length is not 2 ||
+			notation[0] < 'A' || 'H' < notation[0] ||
+			notation[1] < '1' || '8' < notation[1])
+			throw new FormatException($@"{nameof(notation)} ""{notation}"" is not valid");
+		return (notation[0] - 'A', notation[1] - '1');
+	}
+
+	public static bool IsValidPosition(int x, int y) =>
+		0 <= x && x < 8 &&
+		0 <= y && y < 8;
+
+	public (Piece A, Piece B) GetClosestRivalPieces(PieceColor priorityColor)
```

Comment:
again would just suggest following naming standards `Piece a, Piece b`

Reply:
As discussed on discord I prefer Pascal case for value tuple syntax. Thanks for the recommendation though.